packages/api/source/utils/templates.ts

```diff
@@ -0,0 +1,228 @@
+import Mustache from "mustache";
+
+export const invitationTemplate = (data) => {
+    const { token } = data;
+    return Mustache.render(
+        `<!doctype html>
+    <html>
+    
```

Comment:
How many times should I tell? This needs to be in a template file that's going to be loaded using `fs`.

Reply:
If you're going to use string interpolation then why is Mustache necessary? :/

Reply:
Read the documentation for Mustache, man.